geomstats/geometry/manifold.py

```diff
@@ -82,6 +88,30 @@ def equip_with_metric(self, Metric=None, **metric_kwargs):
 
         self.metric = Metric(self, **metric_kwargs)
 
+    def equip_with_group_action(self, group_action):
+        self.group_action = group_action
+
+    def _check_equip_with_quotient_structure(self):
+        for structure_name in ("metric", "group_action"):
+            if not hasattr(self, structure_name):
+                raise ValueError(f"Need to equip with `{structure_name}` first")
+
+    def equip_with_quotient_structure(self):
+        self._check_equip_with_quotient_structure()
+
+        key = type(self.metric), self.group_action
+
+        out = self._quotient_map.get(key, None)
+        if out is None:
+            raise ValueError(f"No mapping for key: {key}")
+        FiberBundle_, Metric = out
```

Comment:
why `FiberBundle_` (with a "_")?

Reply:
just because we have the object `FiberBundle` and `self._quotient_map.get(key, None)` is necessarily giving that object (e.g. `ShapeBundle` for `DiscreteCurves`). `FiberBundle` can work also, but may lead to think it is really `FiberBundle`.